tests/boundingboxcomputation_tests.ts

```diff
@@ -16,8 +13,16 @@ const BoundingBoxComputationTests = {
     QUnit.module('BoundingBoxComputation');
     test('Point Test', point);
     const run = VexFlowTests.runTests;
-    quadraticParams.forEach((params /*, index*/) => run('Quadratic Test', quadratic, params));
-    cubicParams.forEach((params /*, index*/) => run('Cubic Test', cubic, params));
+    let n = 1;
+    quadraticParams.forEach((params /*, index*/) => {
```

Comment:
You should be able to uncomment the `/*, index*/` and use it instead of incrementing `n` manually. Array.forEach passes in the current item index.

Reply:
done